app/components/features/target-spf-item/types.ts

```diff
@@ -0,0 +1,22 @@
+export enum Type {
+  BIOREGIONAL = 'bioregional',
+  SPECIES = 'species',
+  BIOREGIONAL_AND_SPECIES = 'bioregional-and-species',
+}
+
+export interface TargetSPF {
+  isAllTargets: boolean;
+  id: string;
+  targetValue: number;
```

Comment:
let's simplify the name of these attributes
targetValue => target
spfValue => spf